docs/api/collection.md

```diff
@@ -117,6 +126,9 @@ Fetch objects that the current user should see at their root level. As mentioned
   By default, this will show the 'normal' Collections namespace; to view a different Collections namespace, such as
   `snippets`, you can pass the `?namespace=` parameter.
 
+  Note that this endpoint should return results in a similar shape to `/api/dashboard/:id/items`, so if this is
```

Comment:
This seems like an internal note that should be a code comment rather than a docstring